lib/helpers/getDomNodeProfile.js

```diff
@@ -13,23 +15,31 @@ const getTextContent = (el, options) => {
 }
 
 export function getDomNodeProfile(el, options = {}) {
-  return {
-    action: el.action,
+  let profile = {
     class: el.className,
     href: el.href || null,
     id: el.id,
-    method: el.method,
     name: el.name,
-    ng_click: el.getAttribute('ng-click'),
-    ng_model: el.getAttribute('ng-model'),
+    all_attrs:  getDomNodeAttributes(el, /^((?!value).)*$/),
```

Comment:
Can you pull this rx out to a variable and explain what it's checking? Looks odd